code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -229,7 +230,7 @@
 			<ol>
 				<li>Open the Syndicate Contract Uplink program.</li>
 				<li>Here, you can accept a contract, and redeem your TC payments from completed contracts.</li>
-				<li>The payment number shown in brackets is the bonus you'll receive when bringing your target <b>alive</b>. You receive the
+				<li>The payment number shown in brackets is the bonus you'll recieve when bringing your target <b>alive</b>. You recieve the
```

Comment:
receive is the right spelling

Reply:
https://www.merriam-webster.com/dictionary/receive

Reply:
I didn't even change this, think it may be changes with local version and git

Reply:
can you add an edit, i can't webedit